.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,37 @@
+name: Continuous Integration
+on: [push]
+jobs:
+  test:
+    name: Test ${{ matrix.flavor }} ${{ matrix.node }} ${{ matrix.typescript }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        flavor: [1, 2, 3, 4]
+        include:
+          - flavor: 1
+            node: 6
+            typescript: typescript@latest
+          - flavor: 2
+            node: 13
```

Comment:
FWIW, I usually use `stable` to avoid incrementing this over time. The earliest and lastest node versions has served me pretty well across the things I maintain.